src/radical/pilot/agent/scheduler/base.py

```diff
@@ -332,8 +338,10 @@ def _change_slot_states(self, slots, new_state):
         have been allocated or deallocated.  For details on the data structure,
         see top of `base.py`.
         '''
+        # This method needs to change if the DS changes. 
 
```

Comment:
This is not a useful comment.  This method needs to change on a gazillion other cases, and does not need to change on all DS changes...

Reply:
Removed